sims.mk

```diff
@@ -7,7 +7,8 @@
 ### The strings in there leave off the "test-sim" prefix.
 ###
 ### Environment Variables:
```

Comment:
Should we also add SIMAPP?

Reply:
The `SIMAPP` variable is defined with a `=` (not a `?=`), so it's not an environment variable that can be defined outside the makefile.